tests/sentry/issues/endpoints/test_organization_group_index.py

```diff
@@ -4447,26 +4447,3 @@ def test_bulk_delete_performance_issues(self) -> None:
             assert response.status_code == 204
 
         self.assert_deleted_groups(groups)
-
-    @patch("sentry.api.helpers.group_index.delete.may_schedule_task_to_delete_hashes_from_seer")
```

Comment:
I could have changed the behaviour of this test but it seemed trivial to keep a test just to prove that failing to communicate with Seer will keep going (try/except block in `delete_group_hashes`.